integration/compatibility/generate.py

```
@@ -45,8 +45,8 @@ def setup_class(cls):
         if cls.ANSWER_FILE_NAME is None:
             raise NotImplementedError("Subclass must define ANSWER_FILE_NAME")
             
-        if os.system("docker run --rm -d --name Generate-search -p 6380:6379 redis/redis-stack-server") != 0:
-            print("Failed to start Redis Stack server, please check your Docker setup.")
+        if os.system("docker run --rm -d --name Generate-search -p 6380:6379 redis:latest") != 0:
```

Ruff (0.15.20)

[error] Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`

(S605)